### ISSUE TITLE
Unable to install Windows Package Manager via App Installer (maybe it's me)

### ISSUE THREAD

**Issue by PatrickLownds** [microsoft/winget-cli#251]
# Brief description of your issue

# Steps to reproduce

Deploy Windows 10 Preview image from public Azure. Join the Windows Package Manager Insider Program. Follow the on-boarding e-mail instructions i.e. run **ms-windows-store://pdp/?productid=9nblggh4nns1** 

# Expected behavior

Windows Package Manager is installed via App Installer.

# Actual behavior

Windows Package Manager is not installed the command WinGet is not recognised.

# Environment

```none
[winget --info]
Windows Package Manager version: didn't get installed
Windows: Windows: Windows.Desktop v10.0.19041.208
Package: Microsoft.DesktopAppInstaller v1.0.41331.0

Any other software? None.
```

**Comment by smabuk** [comment]
This turned up for me a today but wasn't working yesterday.

It may be taking time to distribute to all of the local stores (I'm in the UK).

**Comment by rajmandalia** [comment]
Did you have to run updates for it to install - same problem here. I uninstalled (Get-AppxPackage Microsoft.DesktopAppInstaller |Remove-AppPackage) and reinstalled from the Store but no dice.

**Comment by TomBrien** [comment]
Same here, after following the instructions I installed App Installer via the command in the email however, there is no launch button on the app page after install and it does not appear on my list of installed apps in the store app.

**Comment by rajmandalia** [comment]
Just like smabuk, it now works for me. The next day, I again uninstalled App Installer and reinstalled from the Windows Store - this time it actually took a few seconds longer to install, so I knew it was actualy doing something.

**Comment by TomBrien** [comment]
Same here just took a while to come through in the store. I feel like the email is being sent before the MSA is added to the preview ring. Perhaps it is possible to change the flow so you get a "We've received your request" email then when the MSA account has been added and given any required propagation time the link email is then sent

**Comment by mareksamaj** [comment]
Perhaps also worth documenting the minimum version of App Installer that has winget included. I joined winget preview flight 24hrs ago and I'm still on App Installer 1.0.40491.0 (UK store) with no winget. (19628 Windows Insider build). Also if there is delay between joining the winget preview and getting the updated App Installer from non-US Microsoft App stores that should be also noted somewhere.

**Comment by abhitrueprogrammer** [comment]
I confirm that no app installer updates are provided to me (and winget command is not recognized) even after roughly 2 days of signing up in windows package manager insider program. I am on version 1.0.32812.0  of app installer and windows 2004 update(not on insider variant),even tried uninstalling and reinstalling app installer to no luck.If Microsoft are no longer admitting new participants please add it in the readme.md file.

**Comment by abhitrueprogrammer** [comment]
Just like OP my problem fixed but it took 1 week ,so maybe a conformation mail is really needed to avoid confusion, just like TomBrien suggested.

**Comment by mloskot** [comment]
I can confirm the same issue, no update of App Installer after submitting for the Windows Insider flight ring, re-signing in to the Store, etc.

The https://docs.microsoft.com/en-us/windows/package-manager/winget/ says to either submit for the _Windows Package Manager Insiders Program_ or the _Windows Insider flight ring_ and perhaps  there is a timeline difference between the two programs, perhaps one gets the update earlier than the other. One way or another, it's all very confusing and no info about the App Installer version number installed/required is not helpful at all.

**Comment by mloskot** [comment]
Update to my https://github.com/microsoft/winget-cli/issues/251#issuecomment-733115463 as I have just managed to get the updated version of the App Installer:

1. Uninstall the App Installer `Get-AppxPackage Microsoft.DesktopAppInstaller |Remove-AppPackage` (thanks @rajmandalia for https://github.com/microsoft/winget-cli/issues/251#issuecomment-632486088)
2. Sign up for the _Windows Insider flight ring_ 
3. Settings> Diagnostics & feedback settings and allow the **Optional diagnostic data** as that seems mandatory due [to run Windows 10 Insider Preview Builds, you must allow us to see a certain amount of diagnostic data](https://docs.microsoft.com/en-us/windows-insider/data-settings).
4. Settings> Microsoft Store > install App Installer (followed with Get Updates just in case)
5. I also did Settings > Windows Insider Programme and disabled Windows Insider builds as I don't want the insider system updates.

![image](https://user-images.githubusercontent.com/80741/100130715-5bc28400-2e83-11eb-8cac-c028f5284a4f.png)

I _suspect_ that the critical part of the procedure is the step 3. that is allow the **Optional diagnostic data** which I had turned off.
I _suspect_ this may also be (silently) required by the _Windows Package Manager Insiders Program_ to get the App Installer previews.
